src/server/routes/enrichment-rest.js

```diff
@@ -219,14 +219,14 @@ enrichmentRouter.post('/visualization', (req, res) => {
  *       properties:
  *         genes:
  *           type: array
- *           description: an array of genes, integers
- *             are interpreted as NCBIGENE
+ *           description: "Input genes as JSON array
+ *                        \n By default, numerical inputs will be interpreted as NCBIGENE notation unless specified, ex HGNC:11998"
  *           example: ["TP53", "111", "AFF4", "111", "11998"]
  *           items:
  *             type: string
  *         target:
  *           type: string
```

Comment:
Shouldn't this be 'targetDb'?

Reply:
The documentation changes for the new parameter names is included in the parameter pull requests